addons/cert-manager/0.10.x/cert-manager-6.yaml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: cert-manager
+  labels:
+    kubeaddons.mesosphere.io/name: cert-manager
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.10.1-6"
+    appversion.kubeaddons.mesosphere.io/cert-manager: "0.10.1"
+    docs.kubeaddons.mesosphere.io/cert-manager: "https://docs.cert-manager.io/en/release-0.10/"
+    values.chart.helm.kubeaddons.mesosphere.io/cert-manager: "https://raw.githubusercontent.com/mesosphere/charts/848f804/staging/cert-manager-setup/values.yaml"
```

Comment:
bump values link too, please

Reply:
hmmm ok, that wasnt changed, but ill do so in another PR.

Reply:
If the values commit hasn't changed there should be no.need for that.